test/controllers/user_category_controller_test.exs

```diff
@@ -1,60 +1,40 @@
 defmodule CodeCorps.UserCategoryControllerTest do
-  use CodeCorps.ApiCase
+  use CodeCorps.ApiCase, resource_name: :user_category
 
-  alias CodeCorps.Repo
-  alias CodeCorps.UserCategory
-
-  @attrs %{}
-
-  defp build_payload, do: %{ "data" => %{"type" => "user-category", "attributes" => %{}}}
+  @invalid_attrs %{}
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
       path = conn |> user_category_path(:index)
-      json = conn |> get(path) |> json_response(200)
+      conn = conn |> get(path)
 
-      assert json["data"] == []
+      assert json_response(conn, 200)["data"] == []
     end
 
     test "filters resources on index", %{conn: conn} do
-      society = insert(:category, name: "Society")
-      technology = insert(:category, name: "Technology")
-      government = insert(:category, name: "Government")
-
-      user = insert(:user)
-      user_category_1 = insert(:user_category, user: user, category: society)
-      user_category_2 = insert(:user_category, user: user, category: technology)
-      insert(:user_category, user: user, category: government)
+      [user_category_1, user_category_2] = insert_pair(:user_category)
```

Comment:
A relatively elegant way to do it would be to just replace the line with 

```Elixir
[user_category_1, user_category_2 | _] = insert_list(3, :user_category)
```